### ISSUE TITLE
The value of DR Loss in RetiaNet

### ISSUE THREAD

**Issue by ys0823** [idstcv/DR_loss#10]
Thanks for your great work!
I follow the DR Loss in RetiaNet in maskrcnn-benchmark, but the classification loss is very small, it is about 0.0005 at first and it is nearly not convergent. But when I use Focal Loss, it is a normal value of 1.0+, I don't know what caused it. Are there any suggestions? My num-classes is 4. @qian-qi @idstcv

**Comment by qian-qi** [comment]
Hi, please check the similar problem in closed issue#2. The first loss is guaranteed to be a large one.

**Comment by ys0823** [comment]
> Hi, please check the similar problem in closed issue#2. The first loss is guaranteed to be a large one.

Thanks, I have the same mistake as issue #2. When I  reverse the code, it seems to be normally.

But I have another question when I use DR LOSS instead of Cross entropy loss in SSD, my input of DR LOSS is the batch of logits of confidence and labels, and I didn't use Hard Negative Mining in SSD, the initial loss of classification is about 3.0+, and the same as RetinaNet, I also don't have normalization term in DR LOSS in my SSD, I use the DR lOSS as a mean batch classification loss by default. Based on this situation, the loss of regression loss can not converge well and the same as DR LOSS. However, both regression loss and classification can decline slowly. But the MAP of my SSD is 0.0002, I don't know what is the kernel problem. @qian-qi

**Comment by qian-qi** [comment]
I don't have the log so I'm not sure about the problem. Maybe it's from the learning rate and you can try different training configurations.
